### ISSUE TITLE
Building Cxx fails with Pkg.add()

### ISSUE THREAD

**Issue by ViralBShah** [Keno/Cxx.jl#287]
I was under the impression that Cxx.jl no longer needs patching the julia build, and `Pkg.build()` will suffice, but this is what I get:

```
julia> Pkg.add("Cxx")
INFO: Installing Cxx v0.0.1
INFO: Building Cxx
writing path.jl file
Tuning for julia installation at: /Users/viral/julia/usr/bin
BuildBootstrap.Makefile:5: /Users/viral/julia/usr/bin/../../Make.user: No such file or directory
make: *** No rule to make target `/Users/viral/julia/usr/bin/../../Make.user'.  Stop.
=================================[ ERROR: Cxx ]=================================


LoadError: failed process: Process(`make -j4 -f BuildBootstrap.Makefile BASE_JULIA_HOME=/Users/viral/julia/usr/bin`, ProcessExited(2)) [2]
while loading /Users/viral/.julia/v0.6/Cxx/deps/build.jl, in expression starting on line 31

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: Cxx had build errors.

 - packages with build errors remain installed in /Users/viral/.julia/v0.6
 - build the package(s) and all dependencies with `Pkg.build("Cxx")`
 - build a single package by running its `deps/build.jl` script

================================================================================
```

**Comment by stevengj** [comment]
See also [this thread](https://groups.google.com/d/msg/julia-users/OUGtIFTOZzU/foSGViqRAQAJ) on the mailing list, which has a patch, though maybe not for the same problem?

**Comment by Gnimuc** [comment]
@stevengj this has already been fixed by 5a1e970a5281c56c90cc2ab2be5b6f213d5d30f0 recently.

**Comment by Keno** [comment]
Should be fixed with the 0.0.2 tag: https://github.com/JuliaLang/METADATA.jl/pull/6286.
